docker/Dockerfile

```diff
@@ -102,13 +104,18 @@ ARG TORCH_BENCH_BRANCH=main
 ARG TORCH_BENCH_COMMIT=${TORCH_BENCH_BRANCH} 
 RUN git clone -b ${TORCH_BENCH_BRANCH} https://github.com/pytorch/benchmark.git && \
     cd benchmark && git checkout ${TORCH_BENCH_COMMIT} && pip install --no-deps -r requirements.txt && \
-    pip install --no-cache Jinja2==3.1.2 markupsafe==2.0.1 beartype==0.15.0 && \
+    pip uninstall botocore -y && \
+    pip install --no-cache Jinja2==3.1.2 markupsafe==2.0.1 beartype==0.15.0 urllib3==1.26.16 && \
+    # Workaround for https://github.com/pytorch/benchmark/issues/1772
+    sed -i '/pyworld/d' torchbenchmark/models/tts_angular/requirements.txt && \
+    sed -i '/torchrec-nightly/d' torchbenchmark/models/dlrm/requirements.txt && \
     python install.py && \
     # Need update numpy version to avoid some models crash
     pip install --upgrade numpy
 ARG BENCH_COMMIT=${PT_COMMIT}
 WORKDIR /workspace/pytorch
 RUN git checkout ${BENCH_COMMIT} benchmarks && \
+    pip install z3-solver diffusers && \
```

Comment:
`z3-solver` for deps issue
![MicrosoftTeams-image](https://github.com/chuanqi129/inductor-tools/assets/76785525/21d3e353-58a4-4ac0-b8b0-a3ad9553037b)
`diffusers` for stable_diffusion models https://github.com/pytorch/benchmark/blob/main/torchbenchmark/models/stable_diffusion_unet/__init__.py

Reply:
`z3-solver` should be the dependency of pytorch, so move it to the dev-base part. For `diffusers`, we still need figure out why it can't install by the model installation.

Reply:
I have checked the reason of diffusers installation issue, refer https://github.com/pytorch/benchmark/blob/main/torchbenchmark/models/stable_diffusion_unet/install.py#L13-L16, that means if we don't pass the `HUGGING_FACE_HUB_TOKEN` before the torchbench model install, the diffusers will not install. We need add a `ARG HUGGING_FACE_HUB_TOKEN` in Dockerfile, and pass the token when the docker image build.

Reply:
Thanks, updated in https://github.com/chuanqi129/inductor-tools/pull/13/commits/6040760026706c09a07682857677b578887a145d